src/main/scala/com/gu/certificate/Magic.scala

```diff
@@ -90,15 +90,20 @@ object Magic extends BouncyCastle with FileHelpers {
     System.err.println(s"installing to IAM")
 
     val iamClient = region.createClient(classOf[AmazonIdentityManagementClient], installCredentialsProvider, null)
-    val uploadResult = iamClient.uploadServerCertificate(
-      new UploadServerCertificateRequest()
-        .withServerCertificateName(s"$safeDomain-exp$expDate")
-        .withPrivateKey(decryptedPem)
-        .withCertificateBody(certificatePem)
-        .withCertificateChain(chainPem)
-    )
-
-    System.err.println(s"successfully installed certificate in IAM as ${uploadResult.getServerCertificateMetadata.getArn}")
+    val certificateUploadRequest = new UploadServerCertificateRequest()
+      .withServerCertificateName(s"$safeDomain-exp$expDate")
+      .withPrivateKey(decryptedPem)
+      .withCertificateBody(certificatePem)
+      .withCertificateChain(chainPem)
+
+    Try {
+      val certificateUploadResult = iamClient.uploadServerCertificate(certificateUploadRequest)
+      val certificateArn = certificateUploadResult.getServerCertificateMetadata.getArn
+      System.err.println(s"successfully installed certificate in IAM as ${certificateArn}")
```

Comment:
very small and was already there, but why `System.err` and not `System.out` when the installation is successful?